study.md

```diff
@@ -30,32 +30,44 @@ Please note:
 In your own words, define what the responsibilities of the model layer are in
 Rails.
 
-```md
-<!-- your answer here -->
-```
+
+The model layer in Rails are Ruby classes. It communicates with the database, as well as store and validates data. It handles domain concepts and business logic. It works with the controller to deliver what the user HTTP-requests.
```

Comment:
Yes, the model layer is made of Ruby classes. As are the other layers. 😄

Good explanation.